tests/wpunit/CommentObjectQueriesTest.php

```diff
@@ -666,7 +666,11 @@ public function dataProviderSwitchUser() {
 	public function testQueryingAvatarOnUserAuthorsIsValidForPublicAndAuthenticatedRequests() {
 
 		// create a comment with a guest author as the author
-		$comment_id = $this->createCommentObject();
+		$comment_by_comment_author_id = $this->createCommentObject([
+			'user_id' => 0
+		]);
+
+		$comment_by_user_id = $this->createCommentObject();
```

Comment:
Might be beyond the scope of this PR, but maybe we should clean up these comments at the end of the test?
Worried it might affect #2408, although if it does, i can always handle it there...

Reply:
ya, good idea. I'm bad at cleaning up after my tests 🤦🏻‍♂️ 

I'll update.